assets/archers_paradox/lang/zh_tw.json

```
@@ -0,0 +1,49 @@
+{
+  "effect.archers_paradox.challenge_complete": "挑戰完成",
+  "effect.archers_paradox.challenge_miss": "挑戰失敗",
+  "effect.archers_paradox.challenge_streak": "弓箭手挑戰",
+
+  "effect.archers_paradox.training_miss": "中斷連擊",
+  "effect.archers_paradox.training_streak": "連擊",
+
+  "itemGroup.archers_paradox": "弓箭手悖論",
+
+  "item.archers_paradox.quiver": "箭袋",
+
+  "item.archers_paradox.blaze_arrow": "烈焰箭矢",
+  "item.archers_paradox.blaze_arrow.desc": "箭矢撞擊時將附近方塊與生物點燃。",
+  "item.archers_paradox.challenge_arrow": "挑戰者箭矢",
+  "item.archers_paradox.challenge_arrow.desc": "不造成傷害，用於測試弓箭手的技巧。讓每一發都更有難度。",
+  "item.archers_paradox.diamond_arrow": "鑽石箭矢",
+  "item.archers_paradox.diamond_arrow.desc": "增加傷害，射穿一個目標",
+  "item.archers_paradox.displacement_arrow": "置換箭矢",
+  "item.archers_paradox.displacement_arrow.desc": "創造一個暫時的區域，將事物傳送走。",
+  "item.archers_paradox.ender_arrow": "終界箭矢",
+  "item.archers_paradox.ender_arrow.desc": "傳送被擊中者到身旁，或傳送到箭矢所落之處。",
+  "item.archers_paradox.explosive_arrow": "爆炸箭矢",
+  "item.archers_paradox.explosive_arrow.desc": "箭矢撞擊時產生爆炸。",
+  "item.archers_paradox.frost_arrow": "冰霜箭矢",
+  "item.archers_paradox.frost_arrow.desc": "箭矢擊中時凍結附近方塊和凍僵目標。",
+  "item.archers_paradox.glowstone_arrow": "螢光石箭矢",
+  "item.archers_paradox.glowstone_arrow.desc": "讓周圍稍微變亮。",
+  "item.archers_paradox.lightning_arrow": "閃電箭矢",
+  "item.archers_paradox.lightning_arrow.desc": "箭矢撞擊時呼喚下雷。",
```

```suggestion
  "item.archers_paradox.lightning_arrow.desc": "箭矢命中時召喚閃電。",
```
箭矢用「命中」應該比較好
`Lightning` 原版是翻 `閃電`，閃電比較常聽到都是用召喚